observability/prometheus/monitors/rabbitmq-servicemonitor.yml

```diff
@@ -18,6 +18,6 @@ spec:
       insecureSkipVerify: true
   selector:
     matchLabels:
-      app.kubernetes.io/component: rabbitmq
+      app.kubernetes.io/component: rabbitmq-operator
```

Comment:
Is this matching label for the operator or rabbitmq instances? If it's rabbitmq clusters, I think it should stay as `app.kubernetes.io/component: rabbitmq`